### DISCUSSION TITLE
How to show  mem/cpu usage & temp modules on waybar?

### DISCUSSION THREAD

**Discussion by 0xArCHDeViL** [prasanthrangan/hyprdots#1249] [General]
Just how?

**Comment by RookiexCookie** [comment]
you can either switch between different waybar layouts with windows+alt+up/down key or you can add them in the config.ctl 
located at `~/.config/waybar/config.ctl`

![image](https://github.com/prasanthrangan/hyprdots/assets/136296017/b871bbc3-3950-4e3a-927a-f50382bb2b0d)

**Reply by 0xArCHDeViL** [reply]
thanks brother.

**Reply by 0xArCHDeViL** [reply]
ah, last question. how do I enlarge the waybar?
You know, the default size of the waybar is very small...
I've tried setting the height in the css but it doesn't change.

**Reply by RookiexCookie** [reply]
In the image do you see those numbers in second column like 28,34
if you change those the bar size will change

**Reply by kRHYME7** [reply]
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/424dacaf-c6c2-4cdb-b687-387cb99e497d)

Represents the scaling of waybar
